ansible/roles/hpctests/tasks/hpl-solo.yml

```diff
@@ -42,8 +42,8 @@
 - debug:
     msg: "Using {{ hpctests_hplsolo_ntasks }} process per node with P={{ hpctests_hplsolo_pq.grid.P }}, Q={{ hpctests_hplsolo_pq.grid.Q }} targeting {{ (hpctests_hpl_mem_frac | float)  * 100 }}% of {{ hpctests_nodeinfo.info['MEMORY'][0] }} MB memory per node, block size (NB) = {{ hpctests_hpl_NB }}, problem size (N) = {{ hpctests_hplsolo_N }}"
 
-- name: Get all nodes
-  shell: "sinfo --Node --noheader --format %N" # TODO: assumes only one partition, although actually excluding nodes not in the default partition should be fine.
+- name: Get all nodes in partition
+  shell: "sinfo --Node --noheader --format %N --partition={{ hpctests_partition }}"
```

Comment:
Does `sinfo` still truncate outputs in certain circumstances?

Reply:
Using `--format` is [safe](https://slurm.schedmd.com/sinfo.html#OPT_format):
> Minimum field size. If no size is specified, whatever is needed to print the information will be used.

Using `--Format` is [not](https://slurm.schedmd.com/sinfo.html#OPT_Format)
> The minimum field size. If no size is specified, 20 characters will be allocated to print the information.